docs/game_sdk/Store.md

```diff
@@ -373,10 +398,16 @@ curl https://discordapp.com/api/v6/applications/461618159171141643/entitlements?
 
 Fetch an entitlement by its ID. This may be useful in confirming that a user has a given entitlement that another call or the SDK says they do.
 
+###### Query Parameters
+
+| name           | type | description                                                                                         |
+| -------------- | ---- | --------------------------------------------------------------------------------------------------- |
+| with_payments? | bool | returns [limited payment data](#DOCS_GAME_SDK_STORE/http-specific-data-models) for each entitlement |
```

Comment:
this is `with_payment` not plural

Reply:
Is it plural on the multiple entitlements endpoint?

Reply:
yes